IABTechLab/uid2-self-serve-portal
# Updates tour for audit trail and participant switcher

What Changed:

- New tour steps added
- Conditionally showing certain tour steps based on user roles
- Updated test files to accommodate for UI tour using current user and current participant
- Audit trail instructions from Gen

Test Plan:
- Change version number from 0.49.0 to 0.48.0 in `tourSteps`
- To test, delete tour-data in local storage
- UID2 support role: should see audit trail step and participant switcher step
- Admin role (w/ multiple participants): should see audit trail step and participant switcher step
- Admin role (w/ single participant): should see audit trail step
- Operations role (w/ multiple participants): should see participant switcher step
- Operations role (w/ single participant): shouldn't see any of the version 0.48 steps
